src/components/elements/List/style.scss

```diff
@@ -0,0 +1,80 @@
+:local {
+  @import '../../../styles/config';
+
+  .List {
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+    list-style: none;
+
+    > li {
```

Comment:
Basically, you are right. However, it cannot be applied 100% cases. I started with BEM for the children `li` element of `List`, then we lost flexibility for styling of `li` from the app since `{children}` cannot has custom classes into it. I tried to add BEM style class to `ListItem` and I cannot have multiple `li` in it since `ListItem` is `li` itself. So, even thought BEM is good for CSS concept, it cannot be applied in this case due to React `{children}` usage.